Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM osgeo/gdal:ubuntu-small-latest
+FROM osgeo/gdal:ubuntu-small-3.5.0
```

Comment:
I think this is not part of this PR?

Reply:
Noo it is not.

Reply:
It will get merged with the other one anyway :)